### DISCUSSION TITLE
New layouts

### DISCUSSION THREAD

**Discussion by ghost** [cotes2020/jekyll-theme-chirpy#1969] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hi, have anyone created any new layouts?
The main layouts we have for any kind of content page are page and post only, and there's also archive, categories, tags, home, page etc...

I'm wondering if there's different kinds of layouts created by other users.
I was searching for ways to do grid layouts that's like YouTube.
Cause I want to embed different youtube videos and playlists.

### Operations you have already tried

Categories layout is the closest of what I want but in list view....... I'm trying to fiddle with that code in my free time, sadly not much time these days.

**Comment by huanyushi** [comment, marked as answer]
You can modify the layout to achieve the functions you want, see [here](https://github.com/cotes2020/jekyll-theme-chirpy/tree/master/_layouts). But you said that the layouts like YouTube requires a UI redesign, I think this is not an easy task, you need to do it yourself.

**Reply by ghost** [reply]
@huanyushi can I make a seperate layout that would work like categories?
I mean, the url is abc.com/categories/cat_name
Can I make it
abc.com/lectures/group1

Like this?

**Reply by ghost** [reply]
Nevermind I found the way after tweaking with permalink structure in layouts and config
